viewing_party/main.py

```diff
@@ -0,0 +1,115 @@
+#WAVE 1 *******************************************************************************************************************************************************
+
+def create_movie(title, genre, rating):
+    while title and genre and rating:
+        return {"title":title, "genre":genre, "rating":rating}
+        
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for header in user_data["watchlist"]:
+        if header["title"] == title:
+            user_data["watchlist"].remove(header)
+            user_data["watched"].append(header)
+    return user_data
+
+
+#WAVE 2 *******************************************************************************************************************************************************
+
+def get_watched_avg_rating(user_data):
+    total = 0
+    if len(user_data["watched"]) == 0:
+        return 0
+    for movie in user_data["watched"]:
+        total += movie["rating"]
+    else:
+        return total/len(user_data["watched"])
```

Comment:
this works just fine! maybe we can use only one return statement:

```suggestion
    total = 0
    if len(user_data["watched"]) > 0:
        for movie in user_data["watched"]:
            total += movie["rating"]
        total = total/len(user_data["watched"])
   return total
```